gef.py

```diff
@@ -3938,17 +3952,17 @@ def get_filename() -> str:
 
 
 @deprecated("Use `gef.heap.main_arena`")
-def get_glibc_arena():  # -> GlibcArena
+def get_glibc_arena() -> GlibcArena:
```

Comment:
Can also be `None` so

```suggestion
def get_glibc_arena() -> Optional[GlibcArena]:
```